packages/@aws-cdk-testing/cli-integ/lib/with-cdk-app.ts

```diff
@@ -135,6 +137,27 @@ export function withDefaultFixture(block: (context: TestFixture) => Promise<void
   //              ^~~~~~ this is disappointing TypeScript! Feels like you should have been able to derive this.
 }
 
+export interface DisableBootstrapContext {
+  /**
+   * Whether to disable creating the default bootstrap
+   * stack prior to running the test
+   *
+   * This should be set to true when running tests that
+   * explicitly create a bootstrap stack
+   *
+   * @default false
+   */
+  readonly disableBootstrap?: boolean;
+}
+
+/**
+ * To be used in place of `withDefaultFixture` when the test
+ * should not create the default bootstrap stack
+ */
+export function withoutBootstrap(block: (context: TestFixture & DisableBootstrapContext) => Promise<void>) {
+  return withAws<TestContext>(withCdkApp(block));
```

Comment:
You forgot to add `disableBootstrap`:

```suggestion
  return withAws<TestContext>(withCdkApp((context) => block({ ...context, disableBootstrap: true })));
```